src/components/users/LoginForm/LoginForm.js

```diff
@@ -1,24 +1,54 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+
 import "style/common/_common.scss";
 import CommonInput from "components/common/CommonInput";
+import { userActions } from "actions/userActions";
+import { userActionTypesConstants } from "constants/userConstants";
 
 import "./login-form.scss";
 
-const LoginForm = () => (
-  <form className="login-form">
-    <CommonInput
-      label="Email"
-      inputType="email"
-      placeHolder="Type your email"
-    />
-    <CommonInput
-      label="Password"
-      inputType="password"
-      placeHolder="Type your password"
-    />
-    <button type="button" className="regular-text">
-      Log in
-    </button>
-  </form>
-);
+const { USER_REQUEST } = userActionTypesConstants;
+
+const LoginForm = () => {
+  const dispatch = useDispatch();
+
+  const [inputs, setInputs] = useState({ email: "", password: "" });
+  const { email, password } = inputs;
+
+  const handleChange = ({ target: { name, value } }) => {
+    setInputs((inputs) => ({ ...inputs, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    dispatch({ type: USER_REQUEST });
+    e.preventDefault();
+    dispatch(userActions.login(email, password));
+  };
+
+  return (
+    <form className="login-form" onSubmit={handleSubmit}>
```

Comment:
It's okay if you want to make vanilla forms but I suggest using some library to manage them. It could be [Formik](https://formik.org/) or [React Hook Form](https://react-hook-form.com/). I personally prefer React Hook Form since it has a more performant way of doing things.

Reply:
Ok I'll add this in another PR